pkg/investigations/chgm/util.go

```diff
@@ -22,7 +23,8 @@ func createEgressSL(blockedUrls string) *ocm.ServiceLog {
 
 func createEgressLS(blockedUrls string) *ocm.LimitedSupportReason {
 	details := fmt.Sprintf("Your cluster requires you to take action. SRE has observed that there have been changes made to the network configuration which impacts normal working of the cluster, including lack of network egress to these internet-based resources which are required for the cluster operation and support: %s. Please revert changes, and refer to documentation regarding firewall requirements for PrivateLink clusters: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/prepare_your_environment/rosa-sts-aws-prereqs#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs#", blockedUrls)
-
+	details = strings.ReplaceAll(details, "(", "'")
```

Comment:
What would the limited support message look like with this?

Reply:
Network verifier reported failure: https://www.redhat.com:443 'Failed to connect to www.redhat.com port 443: Connection timed out',https://console.redhat.com:443 'Failed to connect to console.redhat.com port 443: Connection timed out',https://openshift.org:443 'Failed to connect to openshift.org port 443: Connection timed out',https://infogw.api.openshift.com:443 'Failed to connect to infogw.api.openshift.com port 443: Connection timed out',https://observatorium.api.openshift.com:443 'Failed to connect to observatorium.api.openshift.com port 443: Connection timed out',https://cdn03.quay.io:443 'Failed to connect to cdn03.quay.io port 443: Connection timed out',https://sso.redhat.com:443 'Failed to connect to sso.redhat.com port 443: Connection timed out',https://catalog.redhat.com:443 'Failed to connect to catalog.redhat.com port 443: Connection timed out',https://nosnch.in:443 'Failed to connect to nosnch.in port 443: Connection timed out',https://api.openshift.com:443 'Failed to connect to api.openshift.com port 443: Connection timed out',https://api.pagerduty.com:443 'Failed to connect to api.pagerduty.com port 443: Connection timed out',https://events.pagerduty.com:443 'Failed to connect to events.pagerduty.com port 443: Connection timed out',https://pull.q1w2.quay.rhcloud.com:443 'Failed to connect to pull.q1w2.quay.rhcloud.com port 443: Connection timed out',https://registry.access.redhat.com:443 'Failed to connect to registry.access.redhat.com port 443: Connection timed out',https://registry.connect.redhat.com:443 'Failed to connect to registry.connect.redhat.com port 443: Connection timed out',https://dvbwgdztaeq9o.cloudfront.net:443 'Failed to connect to dvbwgdztaeq9o.cloudfront.net port 443: Connection timed out',https://api.deadmanssnitch.com:443 'Failed to connect to api.deadmanssnitch.com port 443: Connection timed out',https://cdn02.quay.io:443 'Failed to connect to cdn02.quay.io port 443: Connection timed out',https://www.okd.io:443 'Failed to connect to www.okd.io port 443: Connection timed out',https://mirror.openshift.com:443 'Failed to connect to mirror.openshift.com port 443: Connection timed out',https://aws.amazon.com:443 'Failed to connect to aws.amazon.com port 443: Connection timed out',https://http-inputs-osdsecuritylogs.splunkcloud.com:443 'Failed to connect to http-inputs-osdsecuritylogs.splunkcloud.com port 443: Connection timed out',https://observatorium-mst.api.openshift.com:443 'Failed to connect to observatorium-mst.api.openshift.com port 443: Connection timed out',https://quay.io:443 'Failed to connect to quay.io port 443: Connection timed out',https://registry.redhat.io:443 'Connection timed out after 2000 milliseconds',https://cdn01.quay.io:443 'Connection timed out after 2000 milliseconds',https://quay-registry.s3.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://api.access.redhat.com:443 'Connection timed out after 2000 milliseconds',https://rhc4tp-prod-z8cxf-image-registry-us-east-1-evenkyleffocxqvofrk.s3.dualstack.us-east-1.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://oso-rhc4tp-docker-registry.s3-us-west-2.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://quayio-production-s3.s3.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://ec2.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://iam.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://route53.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://sts.amazonaws.com:443 'Connection timed out after 2000 milliseconds',https://sts.us-east-1.amazonaws.com:443 'Connection timed out after 2000 milliseconds',tcp://inputs1.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs2.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs4.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs5.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs6.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs7.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs8.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs9.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs10.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs11.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs12.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs13.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2000 milliseconds',tcp://inputs14.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2017 milliseconds',tcp://inputs15.osdsecuritylogs.splunkcloud.com:9997 'Connection timed out after 2017 milliseconds',https://ec2.us-east-1.amazonaws.com:443 'Connection timed out after 2001 milliseconds',https://elasticloadbalancing.us-east-1.amazonaws.com:443 'Connection timed out after 2001 milliseconds',https://events.us-east-1.amazonaws.com:443 'Connection timed out after 2001 milliseconds',https://tagging.us-east-1.amazonaws.com:443 'Connection timed out after 2001 milliseconds',https://tagging.us-east-1.amazonaws.com:443 'Timeout was reached',https://cert-api.access.redhat.com:443 'Connection timed out after 2000 milliseconds'"

Reply:
Although for the service log its posts the egress urls: https://github.com/openshift/osdctl/blob/281129115b5f444bef9bf859e901f1b49bcc9b47/cmd/network/verification.go#L40C34-L40C153 

https://github.com/openshift/osdctl/blob/281129115b5f444bef9bf859e901f1b49bcc9b47/cmd/network/verification.go#L278

Do we just not run into the issue here because service logs accept `()` but limited support doesn't on the OCM api side?

Reply:
exactly, the correct fix is to allow the limited support api to have parenthesis if  possible